admin-panel/src/main/java/net/cryptic_game/backend/admin/Config.java

```diff
@@ -1,17 +1,15 @@
 package net.cryptic_game.backend.admin;
 
 import lombok.Getter;
-import org.springframework.beans.factory.annotation.Value;
-import org.springframework.stereotype.Component;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
 
 @Getter
-@Component
+@Setter
```

Comment:
```suggestion
@Data
```
Here the use of @Data is better.

Reply:
We don't need the setters, do we?